### ISSUE TITLE
Preprocessor Documentation issue

### ISSUE THREAD

**Issue by a-kest** [chipsalliance/verible#2215]
Hello in issues there wasnt a selection for the preprocessor so I took the closest I found.

**Describe the bug**

In https://github.com/chipsalliance/verible/tree/master/verilog/tools/preprocessor README it says to use the +include+ flag to include directories, however this does not work. Instead +incdir+ should be used I guess.

**Comment by hzeller** [comment]
Yes, looks like a typo. Can you send a pull request fixing it ? It is indeed `+incdir+`.
